test/Multichain.js

```diff
@@ -4,116 +4,131 @@ let expect = chai.expect
 var Multichain = require("../build/transport/Multichain").Multichain
 var MultichainConnection = require("../build/transport/Multichain").MultichainConnection
 var path = require("path")
+const mock = require('./multichain-data/multichainMock.json')
 require("dotenv").config({ path: path.join(__dirname, "..", "build", "test.env") })
 
 describe('Multichain', () => {
+    var multichain
+    before(function () {
+        multichain = makeConnectedMultichainObject()
+    })
+
     it('connects to a valid multichain instance', () => {
-        var multichain = makeConnectedMultichainObject()
-        multichain.Info(function (err, info) {
-            expect(err).to.not.exist
+        multichain.Info(function (error, info) {
+            expect(error).to.not.exist
             expect(info).to.exist
             expect(info.version).to.equal(mock.info.version)
         })
     })
 
     it('allows instantiation without a connection object', () => {
-        var multichain = new Multichain()
-        expect(multichain).to.exist
+        var empty_multichain = new Multichain()
+        expect(empty_multichain).to.exist
```

Comment:
Add assertion(s) re: the expected state of a multichain instance w/ no connection

Reply:
There isn't a ton of information to work from to add additional assertions, unfortunately. I intend to take another look in the morning. For now, I have added a `hasConnection` function to `multichain.ts`, and make use of it in these tests.